src/styles/utilities/mixins.scss

```diff
@@ -0,0 +1,31 @@
+@mixin sideMargin {
+  margin: 0 16px;
+}
+
+@mixin mainColor {
+  color: #060b35;
+}
+
+@mixin secoundaryColor {
+  color: #616070;
+}
+
+@mixin smallFont {
+  font-weight: 400;
+}
+
+@mixin bigFont {
+  font-weight: 700;
+}
+
+@mixin activeStar {
+  flex-direction: row;
+  display: flex;
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  align-items: center;
+  margin-right: 4px;
+  background: url("/images/star-active.svg") no-repeat center center;
+  z-index: 2;
```

Comment:
dont duplicate this styles already in star

Reply:
I needed to, otherwise it didn't work, newly displayed active starts weren't shown.

Reply:
you just need to change background of stars instead of changing z-index